### PR TITLE
enabling g4studyMT

### PR DESCRIPTION
**Pull request by resnegfk**
enabling an option to run g4study in the MT mode

### CODE REVIEW

File: Mu2eG4/src/Mu2eG4MTRunManager.cc [hunk 1]
```diff
@@ -171,8 +172,10 @@ namespace mu2e {
                                            make_unique<ConstructMaterials>(conf_)));
     }
     else {
-      throw cet::exception("CONFIG")
-        << "Error: You are trying to run in MT mode without the Standard Mu2e Detector!\n";
+      allMu2e = (new WorldMaker<Mu2eStudyWorld>(make_unique<Mu2eStudyWorld>(conf_, &sensitiveDetectorHelper_),
+                                           make_unique<ConstructMaterials>(conf_)));
+    //   throw cet::exception("CONFIG")
+    //     << "Error: You are trying to run in MT mode without the Standard Mu2e Detector!\n";
```
Comment (kutschke) [inline Mu2eG4/src/Mu2eG4MTRunManager.cc:178]: Please remove the commented out lines.

**Review by kutschke — CHANGES_REQUESTED** [review]

File: Mu2eG4/src/Mu2eG4MTRunManager.cc (hunk 1 above)
Comment (resnegfk) [reply Mu2eG4/src/Mu2eG4MTRunManager.cc]: Done

**Comment by resnegfk** [comment]
@FNALbuild run build test

**Review by kutschke — APPROVED** [review]
Thanks for making the change.

**Comment by kutschke** [comment]
cosmicOffSpill should be fixed now

@FNALbuild run build test

File: Mu2eG4/src/Mu2eG4MTRunManager.cc (hunk 1 above)
Comment (kutschke) [reply Mu2eG4/src/Mu2eG4MTRunManager.cc]: Thanks

**Pull request merged**